indexer/packages/postgres/src/models/leaderboard-pnl-model.ts

```diff
@@ -0,0 +1,60 @@
+import path from 'path';
+
+import { Model } from 'objection';
+
+import { NumericPattern } from '../lib/validators';
+import UpsertQueryBuilder from '../query-builders/upsert';
+import BaseModel from './base-model';
+
+export default class LeaderboardPNLModel extends BaseModel {
+
+  static get tableName() {
+    return 'leaderboard_pnl';
+  }
+
+  static get idColumn() {
+    return ['subaccountId', 'timeSpan'];
+  }
+
+  static relationMappings = {
+    subaccount: {
+      relation: Model.BelongsToOneRelation,
+      modelClass: path.join(__dirname, 'subaccount-model'),
+      join: {
+        from: 'leaderboard_pnl.subaccountId',
+        to: 'subaccounts.id',
+      },
+    },
+  };
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [
+        'subaccountId',
+        'timeSpan',
+        'pnl',
+        'currentEquity',
+        'rank'],
```

Comment:
```suggestion
        'rank'
      ],
```
Super small nit but this would be preferred